docs/rest-api/concepts/scan-guidance.md

```diff
@@ -34,6 +34,18 @@ https://graph.microsoft.com/v1.0/sites/root/drives
 
 The drive is the starting point for large-scale file activities.  You’ll use these drives to get complete lists of files, connect webhooks for notifications, and use delta query to get sets of changes to items in the drives.
 
+### How to find SharePoint site collections and OneDrive for Business drives
+
+Using Microsoft Graph with Application permissions, you can obtain the full list of site collections and OneDrive for Business drives.  To get this list use the following API call:
```

Comment:
Recommend a small adjustment because the API returns a collection of sites:
"full list of site collections, including sites containing OneDrive for Business drives"